xesmf/tests/test_oceanmodels.py

```diff
@@ -0,0 +1,29 @@
+import cf_xarray  # noqa
+import numpy as np
+import xarray as xr
+
+import xesmf
+
+
+def test_MOM6_to_1x1():
+    """ check regridding of MOM6 to regular lon/lat """
+
+    dataurl = 'http://35.188.34.63:8080/thredds/dodsC/OM4p5/'
```

Comment:
Not sure I like the idea of streaming data from a THREDDS server in the test suite. We're doing this on our end and its prone to breakage, network issues, etc. I suggest creating a similar, small synthetic test file on the fly.

Reply:
fair point! I made that test optional for the CI but I'd like to keep the possibility to do regression tests against real data.